libraries/rush-lib/src/schemas/pnpm-config.schema.json

```diff
@@ -0,0 +1,72 @@
+{
```

Comment:
I expressly do not want this file validated by Rush, so that we don't have to update Rush if pnpm adds more supported fields. End users can simply upgrade their version of pnpm and get the new settings. Ideally that reduces to copying the content of this file directly into `common/temp/package.json` as the `pnpm` property. Pinging @octogonz since I imagine he has the opposite perspective.

Reply:
This config file should be validated by Rush, but the `pnpmFieldInRootPackageJson` property shouldn't be.